### ISSUE TITLE
Read-only: systemd-tmpfiles 

### ISSUE THREAD

**Issue by nettings** [nettings/medianet#5]
Feb 08 14:31:51 mn_29f4ff systemd[1]: Starting Cleanup of Temporary Directories...
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/tr/cat8): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/tr/cat5): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/tr/cat1): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/fr.ISO8859-1/cat8): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/fr.ISO8859-1/cat7): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/ru/cat8): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/ru/cat5): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/ru/cat1): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/fi/cat8): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/fi/cat1): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/da/cat8): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/da/cat5): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/da/cat1): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/cs/cat8): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/cs/cat5): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/cs/cat1): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/cs/cat7): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/sv/cat8): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/sv/cat5): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/sv/cat1): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/pt_BR/cat8): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/pt_BR/cat5): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/pt_BR/cat1): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/zh_TW/cat8): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/zh_TW/cat5): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/zh_TW/cat1): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/hu/cat8): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/hu/cat5): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/hu/cat1): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/id/cat8): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/id/cat5): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/id/cat1): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/oldlocal/cat1): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/fr.UTF-8/cat8): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/fr.UTF-8/cat7): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/sl/cat8): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/sl/cat1): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/zh_CN/cat8): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/zh_CN/cat5): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/zh_CN/cat1): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/gl/cat8): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/ko/cat8): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/ko/cat5): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd-tmpfiles[21173]: rmdir(/var/cache/man/ko/cat1): Read-only file system
Feb 08 14:31:51 mn_29f4ff systemd[1]: Started Cleanup of Temporary Directories.

**Comment by nettings** [comment]
should be fixed in 7080f89d1b47a6c7b8f937f175aebb71273ba004

**Comment by nettings** [comment]
it's back after introducing the new writable /local partition and changing things around:
```
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/oldlocal/cat1): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: unlink(/var/cache/man/pl/index.db): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/pl/cat1): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/pl/cat8): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/pl/cat5): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: utimensat(/var/cache/man/pl): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: unlink(/var/cache/man/fi/index.db): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/fi/cat1): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/fi/cat8): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: utimensat(/var/cache/man/fi): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: unlink(/var/cache/man/es/index.db): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/es/cat1): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/es/cat8): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/es/cat5): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: utimensat(/var/cache/man/es): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/de/cat1): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/de/cat8): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/de/cat5): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: unlink(/var/cache/man/tr/index.db): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/tr/cat1): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/tr/cat8): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/tr/cat5): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: utimensat(/var/cache/man/tr): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/cat2): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/cat1): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: unlink(/var/cache/man/fr.UTF-8/index.db): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/fr.UTF-8/cat8): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/fr.UTF-8/cat7): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: utimensat(/var/cache/man/fr.UTF-8): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: unlink(/var/cache/man/sl/index.db): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/sl/cat1): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/sl/cat8): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: utimensat(/var/cache/man/sl): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: unlink(/var/cache/man/da/index.db): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/da/cat1): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/da/cat8): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/da/cat5): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: utimensat(/var/cache/man/da): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: unlink(/var/cache/man/zh_TW/index.db): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/zh_TW/cat1): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/zh_TW/cat8): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/zh_TW/cat5): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: utimensat(/var/cache/man/zh_TW): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: unlink(/var/cache/man/fr.ISO8859-1/index.db): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/fr.ISO8859-1/cat8): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/fr.ISO8859-1/cat7): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: utimensat(/var/cache/man/fr.ISO8859-1): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: unlink(/var/cache/man/pt/index.db): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/pt/cat1): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/pt/cat8): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/pt/cat5): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: utimensat(/var/cache/man/pt): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/cat4): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: unlink(/var/cache/man/fr/index.db): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/fr/cat1): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/fr/cat8): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/fr/cat5): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: utimensat(/var/cache/man/fr): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: unlink(/var/cache/man/id/index.db): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/id/cat1): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/id/cat8): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/id/cat5): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: utimensat(/var/cache/man/id): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: unlink(/var/cache/man/ru/index.db): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/ru/cat1): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/ru/cat8): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/ru/cat5): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: utimensat(/var/cache/man/ru): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/cat8): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: unlink(/var/cache/man/ko/index.db): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/ko/cat1): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/ko/cat8): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/ko/cat5): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: utimensat(/var/cache/man/ko): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: unlink(/var/cache/man/it/index.db): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/it/cat1): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/it/cat8): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/it/cat5): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: utimensat(/var/cache/man/it): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: unlink(/var/cache/man/nl/index.db): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/nl/cat1): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/nl/cat8): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/nl/cat5): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: utimensat(/var/cache/man/nl): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: unlink(/var/cache/man/cs/index.db): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/cs/cat1): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/cs/cat8): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/cs/cat7): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/cs/cat5): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: utimensat(/var/cache/man/cs): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: unlink(/var/cache/man/sv/index.db): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/sv/cat1): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/sv/cat8): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/sv/cat5): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: utimensat(/var/cache/man/sv): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: unlink(/var/cache/man/hu/index.db): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/hu/cat1): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/hu/cat8): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/hu/cat5): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: utimensat(/var/cache/man/hu): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/cat7): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/cat6): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/cat5): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/cat3): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: unlink(/var/cache/man/pt_BR/index.db): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/pt_BR/cat1): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/pt_BR/cat8): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/pt_BR/cat5): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: utimensat(/var/cache/man/pt_BR): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: unlink(/var/cache/man/ja/index.db): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/ja/cat1): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/ja/cat8): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/ja/cat5): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: utimensat(/var/cache/man/ja): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: unlink(/var/cache/man/gl/index.db): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/gl/cat8): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: utimensat(/var/cache/man/gl): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: unlink(/var/cache/man/vi/index.db): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/vi/cat1): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/vi/cat7): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: utimensat(/var/cache/man/vi): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: unlink(/var/cache/man/zh_CN/index.db): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/zh_CN/cat1): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/zh_CN/cat8): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: rmdir(/var/cache/man/zh_CN/cat5): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd-tmpfiles[4348]: utimensat(/var/cache/man/zh_CN): Read-only file system
May 12 18:30:06 mn-VP200A-R systemd[1]: Started Cleanup of Temporary Directories.
```
